src/FffGcodeWriter.h

```diff
@@ -264,12 +264,13 @@ class FffGcodeWriter : public NoCopy
      * This function should be called for only one layer;
      * calling it for multiple layers results in the skirt/brim being printed on multiple layers.
      * 
-     * \param storage where the slice data is stored.
+     * \param for_support Whether or not this is brim specially made underneath support-infill.
+     * \param skirt_brim The outline of the polygons that'll be skirt/brim.
      * \param gcodeLayer The initial planning of the g-code of the layer.
      * \param extruder_nr The extruder train for which to process the skirt or
      * brim.
      */
-    void processSkirtBrim(const SliceDataStorage& storage, LayerPlan& gcodeLayer, unsigned int extruder_nr) const;
+    void processSkirtBrim(const bool for_support, const Polygons& skirt_brim, LayerPlan& gcodeLayer, unsigned int extruder_nr) const;
```

Comment:
Same as below, I'd make `bool for_support = false` and put it at the end of the parameter list.

Reply:
Done in https://github.com/Ultimaker/CuraEngine/pull/1024/commits/5748ac125f8e025874de29e41c4fab96a18e89ff